modules/ROOT/pages/clauses/where.adoc

```diff
@@ -338,6 +338,35 @@ The `name` and `age` for `Peter` are are returned because his name contains "ete
 |===
 
 
+[[match-string-is-normalized]]
+=== Checking if a `STRING` `IS NORMALIZED`
+
+The `IS NORMALIZED` operator is used to check whether the given `STRING` is in the `NFC` Unicode normalization form:
```

Comment:
same comment about `STRING` - usually just written as "string" on this page

Reply:
I will fix this in a separate PR (have carded it) - it should be `STRING` as per the new formatting style. This page in general is in a pretty dire need of restructuring, so will try to get to it sooner rather than later. I think @gem-neo4j PR can go in as is for now.